src/Host/Broker/Impl/Sessions/Session.cs

```diff
@@ -160,6 +161,18 @@ public class Session {
             HostToClientWorker(_process.StandardOutput.BaseStream, hostEnd).DoNotWait();
         }
 
+        private void StartSession() {
+            _process.Start();
+            _process.WaitForExit(250);
+            if (_process.HasExited && _process.ExitCode != 0) {
+                var message = ErrorCodeConverter.MessageFromErrorCode(_process.ExitCode);
```

Comment:
I am looking at https://msdn.microsoft.com/en-us/library/cc704588.aspx. I guess just filtering for negatives may be sufficient.